content/docs/reference/pkg/nodejs/pulumi/pulumi/_index.md

```diff
@@ -2711,44 +2725,44 @@ Inserts new elements at the start of an array.
 Returns an iterable of values in the array
 
 <h4 class="pdoc-member-header" id="UnwrappedArray-Array">
-<a class="pdoc-child-name" href="https://github.com/pulumi/pulumi/blob/e78abcb5b5ce42737735e571d6b82c2dded72b2f/sdk/nodejs/output.ts#L787">property <b>Array</b></a>
+<a class="pdoc-child-name" href="https://github.com/pulumi/pulumi/blob/4cb43b5a1df4de0cd24bf29f622c336df05cdce6/sdk/nodejs/output.ts#L787">property <b>Array</b></a>
 </h4>
 
 <pre class="highlight"><code><span class='kd'></span>Array: ArrayConstructor;</code></pre>
 <h4 class="pdoc-member-header" id="UnwrappedArray-length">
-<a class="pdoc-child-name" href="https://github.com/pulumi/pulumi/blob/e78abcb5b5ce42737735e571d6b82c2dded72b2f/sdk/nodejs/output.ts#L787">property <b>length</b></a>
+<a class="pdoc-child-name" href="https://github.com/pulumi/pulumi/blob/4cb43b5a1df4de0cd24bf29f622c336df05cdce6/sdk/nodejs/output.ts#L787">property <b>length</b></a>
 </h4>
 
 <pre class="highlight"><code><span class='kd'></span>length: <span class='kd'><a href='https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number'>number</a></span>;</code></pre>
 
 Gets or sets the length of the array. This is a number one higher than the highest element defined in an array.
 
 <h3 class="pdoc-module-header" id="UnwrappedObject" data-link-title="UnwrappedObject">
-    <a href="https://github.com/pulumi/pulumi/blob/e78abcb5b5ce42737735e571d6b82c2dded72b2f/sdk/nodejs/output.ts#L789">
+    <a href="https://github.com/pulumi/pulumi/blob/4cb43b5a1df4de0cd24bf29f622c336df05cdce6/sdk/nodejs/output.ts#L789">
         type <strong>UnwrappedObject</strong>
     </a>
 </h3>
 
 <pre class="highlight"><code><span class='kd'>type</span> UnwrappedObject = { ... };</code></pre>
 <h3 class="pdoc-module-header" id="UnwrapSimple" data-link-title="UnwrapSimple">
-    <a href="https://github.com/pulumi/pulumi/blob/e78abcb5b5ce42737735e571d6b82c2dded72b2f/sdk/nodejs/output.ts#L772">
+    <a href="https://github.com/pulumi/pulumi/blob/4cb43b5a1df4de0cd24bf29f622c336df05cdce6/sdk/nodejs/output.ts#L772">
         type <strong>UnwrapSimple</strong>
     </a>
 </h3>
 
-<pre class="highlight"><code><span class='kd'>type</span> UnwrapSimple = UnwrapSimple<T>;</code></pre>
+<pre class="highlight"><code><span class='kd'>type</span> UnwrapSimple = T extends primitive ? T : T extends <a href='/docs/reference/pkg/nodejs/pulumi/pulumi/#Resource'>Resource</a> ? T : T extends <a href='https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array'>Array</a>&lt;infer U&gt; ? UnwrappedArray&lt;U&gt; : T extends <span class='kd'><a href='https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object'>object</a></span> ? UnwrappedObject&lt;T&gt; : <span class='kd'><a href='https://www.typescriptlang.org/docs/handbook/basic-types.html#never'>never</a></span>;</code></pre>
```

Comment:
I wonder if we should omit this type? This is the sort of type signature that can give people people unfamiliar with TypeScript pause (or cause for concern).